### PR TITLE
New changes

### PR DESCRIPTION
**Pull request by mdsaizraja**

### CODE REVIEW

**Comment by mdsaizraja** [comment]
Please find the attached images related to changes.

Screenshot 2022-08-05 at 11 05 48 AM
* Added the FAQ page.
Screenshot 2022-08-05 at 11 06 15 AM
* Worked on responsiveness.
Screenshot 2022-08-05 at 11 06 57 AM
* Added the Blog page.
Screenshot 2022-08-05 at 11 08 09 AM
* Build some components like SubscribeCard and blog.
* Resolved console warnings.

File: src/components/ Enquiry form/ enquiryForm.js [hunk 1]
```diff
@@ -0,0 +1,200 @@
+import React, { useState } from "react";
+import styled, { createGlobalStyle, css } from "styled-components";
+import { Buttondiv } from "../Contact/ContactStyle";
+
+const GlobalStyle = createGlobalStyle`
```
Comment (Nishant88277) [inline src/components/ Enquiry form/ enquiryForm.js]: dont use createGlobalStyle here as this is not a global style sheet

File: src/components/ Enquiry form/ enquiryForm.js [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+  message: "",
+};
+
+function App() {
+  const [state, setState] = useState(initalState);
+  const [error, setError] = useState("");
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log("submitted!");
```
Comment (Nishant88277) [inline src/components/ Enquiry form/ enquiryForm.js]: remove all consols

File: src/components/CompanyProgressNoCard/companyProgressCard.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { createGlobalStyle } from "styled-components";
+
+const GlobalCss = createGlobalStyle`
```
Comment (Nishant88277) [inline src/components/CompanyProgressNoCard/companyProgressCard.js]: dont use createGlobalStyle as this is not a global css

File: src/components/DetailsText.js/detailsText.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
+import React from "react";
+
+import styled, { createGlobalStyle } from "styled-components";
+
+const GlobalStyleText = createGlobalStyle`
```
Comment (Nishant88277) [inline src/components/DetailsText.js/detailsText.js]: dont use createGlobalStyle as this is not a global css

**Pull request merged**